lib/theme-options/options-fundraising.php

```diff
@@ -222,11 +222,17 @@ function nm_register_fundraising_options_metabox() {
   ) );
 
   $main_options->add_field( array(
+    'name' => __( 'CTA Headline', 'NM' ),
+    'desc' => 'Add headline.',
```

Comment:
lets be more descriptive for end users. Explain what happens when this isn't set: that there is a fallback from the normal support section title